batch/collection.go

```diff
@@ -69,7 +70,7 @@ type UpdateCollectionResponse struct {
 func (a *ApiClient) GetCollection(collectionID string) (*ReadCollectionResponse, diag.Diagnostics) {
 	var diags diag.Diagnostics
 
-	body, moreDiags := a.Request("GET", fmt.Sprintf("/collection/%s", collectionID), nil)
+	body, moreDiags := a.Request(http.MethodGet, fmt.Sprintf("/v1/collection/%s", collectionID), nil)
```

Comment:
👍🏼  I dig the use of `http.Method*` here in place of the string values.

Reply:
can't misspell constants 🕺